manifests/rules/out/icmp.pp

```diff
@@ -11,7 +11,7 @@
 ) {
   if $v4_types {
     $v4_types.each | String $icmp_type | {
-      nftables::rule { 'default_out-accept_icmpv4':
+      nftables::rule { "default_out-accept_icmpv4_${regsubst(split($icmp_type, ' ')[0], '-', '_', 'G')}":
```

Comment:
can you maybe add a comment in the code that roughly explains the code / why this is needed?